specification/synapse/data-plane/Microsoft.Synapse/preview/2019-06-01-preview/entityTypes/Trigger.json

```diff
@@ -335,12 +335,55 @@
           "Microsoft.Storage.BlobDeleted"
         ],
         "x-ms-enum": {
-          "name": "BlobEventType",
+          "name": "BlobEventTypes",
```

Comment:
>BlobEventTypes [](start = 19, length = 14)

Can we keep the previous `BlobEventType` ? I made that change before to fix the CI pipeline issue:

```
"readme":"synapse/data-plane/readme.md",
"tag":"package-artifacts-2019-06-01-preview",
"details":"Duplicate Schema named BlobEventTypes -- type: undefined => "string",
items: "$ref":"#/components/schemas/BlobEventTypes" =>,
enum: undefined => ["Microsoft.Storage.BlobCreated",
"Microsoft.Storage.BlobDeleted"],
x-ms-enum: undefined => "name":"BlobEventTypes",
"modelAsString":true ; This error can be temporarily avoided by using the 'modelerfour.lenient-model-deduplication' setting. NOTE: This setting will be removed in a future version of @autorest/modelerfour; schemas should be updated to fix this issue sooner than that."
```

Reply:
done